spec/services/sign_in/session_creator_spec.rb

```diff
@@ -118,7 +118,7 @@
           session = subject.session
           expect(session.handle).to eq(expected_handle)
           expect(session.hashed_refresh_token).to eq(expected_double_hashed_parent_refresh_token)
-          expect(session.refresh_creation).to eq(expected_created_time)
+          expect(session.refresh_creation.to_f.truncate(3)).to eq(expected_created_time.to_f.truncate(3))
```

Comment:
See comment in `udpate_user_spec.rb`

Reply:
oof this is annoying. 

My only feedback here, could you do the truncation in the `expected_created_time` definition itself? So something like `let(:expected_created_time) { Time.zone.now.round(3) }`, that way we'd preserve this expectation and not have to truncate here?

Reply:
yep yep

Reply:
So I just went to rounding across the board so that we can stick with a `ActiveSupport::TimeWithZone` object instead of converting to a float (`#truncate` isn't a method on `TimeWithZone`). Even if the DB is truncating at 6, rounding at 3 shouldn't create any testing irregularities.